app/models/discussion.rb

```diff
@@ -77,7 +77,7 @@ def followers
       active.
       joins("LEFT OUTER JOIN discussion_readers dr ON (dr.user_id = users.id AND dr.discussion_id = #{id})").
       joins("LEFT OUTER JOIN memberships m ON (m.user_id = users.id AND m.group_id = #{group_id})").
-      where('dr.following = TRUE OR (dr.following IS NULL AND m.following_by_default = TRUE)')
+      where('dr.volume = :email OR (dr.volume IS NULL AND m.volume = :email)', { email: DiscussionReader.volumes[:email] })
```

Comment:
Are you super sure you want to call membership.volume that? I feel like it might make it clearer if its default_volume on membership